bench-plotter/src/bench_plotter/plotting/dashboard_processor.py

```
@@ -0,0 +1,1070 @@
+        output_path: Optional output path (if None, generates from safe_panel_title and safe_legend)
+        plot_title: Optional plot title (if None, generates from panel_title and legend_format)
+    """
+    temp_series = []
+    for series_data in runs_data:
+        interval_mode = series_data.get('interval_mode', 'unknown')
+        label = interval_mode
+        timestamps = series_data.get('timestamps', [])
+        values = series_data.get('values', [])
+        try:
+            ws = calculate_optimal_window_size(timestamps) if timestamps else None
+        except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)